src/chat/channel.py

```diff
@@ -51,13 +51,13 @@ def __init__(self, lobby, name, private=False, *args, **kwargs):
         BaseClass.__init__(self, lobby, *args, **kwargs)
 
         self.setupUi(self)
-        
+
         #Special HTML formatter used to layout the chat lines written by people
         self.lobby = lobby
         self.chatters = {}
-        
+
```

Comment:
While I agree, you can ignore whitespaces in any decent diff editor.
Even on github: https://github.com/FAForever/client/pull/265/files?w=1